test/aws/scaleup.yml

```diff
@@ -37,11 +37,17 @@
   - name: Copy ops-mirror.pem
     copy:
       src: ../../inventory/dynamic/injected/ops-mirror.pem
-      dest: /var/lib/yum/ops-mirror.pem
+      dest: /var/lib/dnf/ops-mirror.pem
```

Comment:
This file can be placed almost anywhere btw, https://github.com/openshift/release/blob/master/cluster/test-deploy/aws-4.0/vars.yaml#L20 needs to be changed along with it